libraries/Filter/HarmonicNotchFilter.cpp

```diff
@@ -90,6 +90,13 @@ const AP_Param::GroupInfo HarmonicNotchFilterParams::var_info[] = {
     // @RebootRequired: True
     AP_GROUPINFO("OPTS", 8, HarmonicNotchFilterParams, _options, 0),
 
+    // @Param: FM_RAT
+    // @DisplayName: Throttle notch min freqency ratio
+    // @Description: The minimum ratio below the configured frequency to take throttle based notch filters when flying at a throttle level below the reference throttle. Note that lower frequency notch filters will have more phase lag. If you want throttle based notch filtering to be effective at a throttle up to 30% below the configured notch frequenct then set this parameter to 0.7.
+    // @Range: 0.1 1.0
+    // @User: Advanced
+    AP_GROUPINFO("FM_RAT", 9, HarmonicNotchFilterParams, _freq_min_ratio, 1.0),
```

Comment:
I'm really not big on this parameter name. Maybe INS_HNTCH_MINR ? Also I think there is a reasonable discussion to be had about whether it would be easier for users to set the min *frequency* rather than the min ratio - might be more understandable.

Reply:
@lthall chose the name, I imaging every dev will want a different name

Reply:
I would also vote for a frequency rather than a ratio, its more tangible. Maybe value of 0 is the same as now? I can see why a ratio makes a nice default tho.

Reply:
we really don't want another frequency, it is just too easy to muck up. The BW parameter should have been a ratio too, as that is what it really is - and users muck up setting it in Hz